Common subdirectories: a/docs and b/docs
Common subdirectories: a/examples and b/examples
diff -U 3 a/HaXml.cabal b/HaXml.cabal
--- a/HaXml.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/HaXml.cabal	2009-11-24 22:00:34.000000000 +0100
@@ -52,7 +52,7 @@
         Text.XML.HaXml.Xtract.Lex,
         Text.XML.HaXml.Xtract.Parse
   hs-source-dirs: src
-  build-depends: base, haskell98, polyparse==1.2, filepath
+  build-depends: base, haskell98, polyparse <= 1.3, filepath
   if flag(splitBase)
     build-depends: base >= 3, pretty, random, containers
   else
Common subdirectories: a/script and b/script
Common subdirectories: a/src and b/src
